hibernate-core/src/test/java/org/hibernate/test/hqlfetchscroll/HQLScrollFetchTest.java

```
@@ -276,7 +276,7 @@ public void testScrollOrderChildrenDesc() {
 			fail( "should have failed because data is ordered incorrectly." );
 		}
 		catch ( AssertionError ex ) {
-			// expected
+			throw ex;
```

I think this is not correct, probably we should throw the error only if the message is "should have failed because data is ordered incorrectly." and the same for the `testListOrderChildrenDesc` test.

---

@dreab8 Hi Andrea, the re-throw depends on the error message now. Could you review/approve, please?

---

it looks good to me